src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -391,28 +391,18 @@
      *
      * <p>For example:</p>
      * <pre>
-     * public class MyClass
-     *   implements Serializable
-     * {
-     * }
+     * public class Test {}
```

Comment:
please move right curly braces on the next line

Reply:
ok

Reply:
@timurt done